docs/source/user_guide/configuration_options.rst

```diff
@@ -62,6 +62,27 @@ Log level
       - Error
       - string
 
+Add payload debuging using `RECEPTOR_PAYLOAD_DEBUG=int` envorment variable and using log level debug.
```

Comment:
The Env var is called `RECEPTOR_PAYLOAD_TRACE_LEVEL` not this

Reply:
In general I think it would be a good idea to replace everywhere where you have the word "debug" with the word "trace"